concrete/src/User/User.php

```diff
@@ -498,15 +498,12 @@ public function logout($hard = true)
         $this->unloadAuthenticationTypes();
 
         $this->invalidateSession($hard);
-        $app->singleton(User::class, function () {
+        $app->singleton(User::class, static function () {
```

Comment:
Let's save some memory by telling PHP to ignore the current context of the closure.